server/storage/gdrive.go

```diff
@@ -194,7 +194,7 @@ func (s *GDrive) Head(ctx context.Context, token string, filename string) (conte
 }
 
 // Get retrieves a file from storage
-func (s *GDrive) Get(ctx context.Context, token string, filename string) (reader io.ReadCloser, contentLength uint64, err error) {
+func (s *GDrive) Get(ctx context.Context, token string, filename string, _ *Range) (reader io.ReadCloser, contentLength uint64, err error) {
```

Comment:
you can try making a call to https://pkg.go.dev/google.golang.org/api/drive/v3#FilesGetCall.Header
adding the proper range at https://github.com/dutchcoders/transfer.sh/blob/2bda0a1e554acea101f771322c28ee63c5762620/server/storage/gdrive.go#L216-L220

```
	fileGetCall := s.service.Files.Get(fileID)
	header := fileGetCall.Header()
	header.Set("Range", rng.Range())
	var res *http.Response
	res, err = fileGetCall.Context(ctx).Download()
	if err != nil {
		return
	}
```